subtests/docker_cli/run/run_tmpfs.py

```diff
@@ -0,0 +1,28 @@
+from run import run_base
+
+
+class run_tmpfs(run_base):
+    # Verify behavior when multiple --name options passed
+
+    def init_subargs(self):
+        super(run_tmpfs, self).init_subargs()
+        self.sub_stuff['expected_option'] = \
+            self.config['run_options_csv'].split('--tmpfs ')[1]
```

Comment:
Python .ini files support string interpolation. The code above is fragile: it will fail, e.g., if run_options_csv is converted to actual CSV (which it should be). You can actually get rid of this entire method by changing run.ini into something like:

    tmpfs_path = /run    # or, even better, /SomEthIngUnlikeLY2appearInNature
    run_options_csv = --detach,--tmpfs,%(tmpfs_path)s

...and using `self.config['tmpfs_path']` in `postprocess()`.